doc/source/whatsnew/v0.18.2.txt

```diff
@@ -19,6 +19,69 @@ Highlights include:
 New features
 ~~~~~~~~~~~~
 
+.. _whatsnew_0182.enhancements.asof_merge:
+
+``pd.merge_asof()`` for asof-style time-series joining
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A long-time requested feature has been added thru the ``pd.merge_asof()`` function, to
+support asof style joining of time-series. (:issue:`1870`). Full documentaion are
+:ref:`here <mergeing.merge_asof>`
+
+Typically we have ``trades`` and ``quotes`` and we want to ``asof-join`` them.
```

Comment:
Also, can you add the explanation what it exactly does (as it is now explained in the docstring and the docs) here as well?

```
Perform an asof merge. This is similar to a left-join except that we match on nearest
key rather than equal keys.
...

```